skyportal/facility_apis/swift.py

```diff
@@ -150,6 +244,131 @@ def _build_payload(self, request):
         return myReq
 
 
+class UVOTXRTBATDataRequest:
+
+    """A JSON structure for Swift UVOT/XRT/BAT Data requests."""
+
+    def __init__(self, request):
+        """Initialize UVOT/XRT/BAT data request.
+
+        Parameters
+        ----------
+        request : skyportal.models.FollowupRequest
+            The request to add to the queue and the SkyPortal database.
+
+        """
+
+        self.requestgroup = self._build_payload(request)
+
+    def _build_payload(self, request):
```

Comment:
I like this because it follows the construction of all the others, but this was mostly heritage from the original ones so might go through and change them all in a future PR.